pkgs/os-specific/darwin/moltenvk/default.nix

```diff
@@ -96,100 +83,107 @@ stdenvNoCC.mkDerivation (finalAttrs: {
+      -lspirv-cross-glsl"
+
     pushd MoltenVKShaderConverter
-      /usr/bin/xcodebuild build \
+      xcodebuild build \
         -jobs $NIX_BUILD_CORES \
-        -derivedDataPath "$derived_data_path" \
         -configuration Release \
         -project MoltenVKShaderConverter.xcodeproj \
         -scheme MoltenVKShaderConverter \
         -arch ${stdenv.targetPlatform.darwinArch}
+      declare -A products=( [MoltenVKShaderConverter]=bin [libMoltenVKShaderConverter.a]=lib )
+      for product in "''${!products[@]}"; do
+        cp MoltenVKShaderConverter-*/Build/Products/Release/$product "$build/''${products[$product]}/$product"
+      done
     popd
-    mkdir -p outputs/bin outputs/lib
-    declare -A outputs=( [MoltenVKShaderConverter]=bin [libMoltenVKShaderConverter.a]=lib )
-    for output in "''${!outputs[@]}"; do
-      cp DerivedData/Build/Products/Release/$output "outputs/''${outputs[$output]}/$output"
-    done
+
+    NIX_LDFLAGS+=" \
+      -lobjc \
+      -lMoltenVKShaderConverter \
+      -lspirv-cross-reflect"
 
     pushd MoltenVK
-      /usr/bin/xcodebuild build \
+      xcodebuild build \
         -jobs $NIX_BUILD_CORES \
-        -derivedDataPath "$derived_data_path" \
         -configuration Release \
         -project MoltenVK.xcodeproj \
         -scheme MoltenVK-macOS \
         -arch ${stdenv.targetPlatform.darwinArch}
+      cp MoltenVK-*/Build/Products/Release/dynamic/libMoltenVK.dylib "$build/lib/libMoltenVK.dylib"
     popd
-    cp DerivedData/Build/Products/Release/dynamic/libMoltenVK.dylib outputs/lib/libMoltenVK.dylib
   '';
 
   installPhase = ''
     mkdir -p "$out/lib" "$out/share/vulkan/icd.d" "$bin/bin" "$dev/include/MoltenVK"
-    cp outputs/bin/MoltenVKShaderConverter "$bin/bin/"
-    cp outputs/lib/libMoltenVK.dylib "$out/lib/"
+    cp build/bin/MoltenVKShaderConverter "$bin/bin/"
+    cp build/lib/libMoltenVK.dylib "$out/lib/"
     cp MoltenVK/MoltenVK/API/* "$dev/include/MoltenVK"
-    ${cctools}/bin/install_name_tool -id "$out/lib/libMoltenVK.dylib" "$out/lib/libMoltenVK.dylib"
-    # FIXME: https://github.com/NixOS/nixpkgs/issues/148189
-    /usr/bin/codesign -s - -f "$out/lib/libMoltenVK.dylib"
     install -m644 MoltenVK/icd/MoltenVK_icd.json "$out/share/vulkan/icd.d/MoltenVK_icd.json"
     substituteInPlace $out/share/vulkan/icd.d/MoltenVK_icd.json \
       --replace ./libMoltenVK.dylib "$out/lib/libMoltenVK.dylib"
   '';
 
-  sandboxProfile = ''
-    (allow file-read* file-write* process-exec mach-lookup)
-    ; block homebrew dependencies
-    (deny file-read* file-write* process-exec mach-lookup (subpath "/usr/local") (with no-log))
+  postFixup = ''
+    install_name_tool -id "$out/lib/libMoltenVK.dylib" "$out/lib/libMoltenVK.dylib"
+    codesign -s - -f "$out/lib/libMoltenVK.dylib"
   '';
 
   meta = {
-    description = "A Vulkan Portability implementation built on top of Apple’s Metal API";
+    description =
+      "A Vulkan Portability implementation built on top of Apple’s Metal API";
```

Comment:
```suggestion
    description = "Vulkan Portability implementation built on top of Apple’s Metal API";
```

Reply:
Not sure how that got introduced. I changed it back. I ran `nixpkgs-fmt`, but it’s apparently not very opinionated on this and accepted it either way.